image/templates/helm/shared/config-templates/scanner-v4/matcher-config.yaml.tpl

```diff
@@ -31,6 +31,7 @@ matcher:
     password_file: /run/secrets/stackrox.io/secrets/password
   vulnerabilities_url: https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version=ROX_VULNERABILITY_VERSION
   indexer_addr: scanner-v4-indexer.{{ .Release.Namespace }}.svc:8443
+  readiness: vulnerability
```

Comment:
Not sure if this simplify things, but you can also set this using env vars: `SCANNER_V4_MATCHER_READINESS=vulnerability`

Reply:
In my latest round of changes, I removed this config, and the tests seemed to pass.  (Check out the description for how I got it to pass without this change.)  I talked with @RTann last week, and he talked me out of setting this option :)